src/cpp/desktop/DesktopWebPage.cpp

```diff
@@ -72,17 +72,12 @@ WebPage::WebPage(QUrl baseUrl, QWidget *parent, bool allowExternalNavigate) :
    defaultSaveDir_ = QDir::home();
    connect(this, SIGNAL(windowCloseRequested()), SLOT(closeRequested()));
    connect(profile(), &QWebEngineProfile::downloadRequested, onDownloadRequested);
+   connect(profile(), &WebProfile::urlIntercepted, this, &WebPage::onUrlIntercepted);
 }
 
 void WebPage::setBaseUrl(const QUrl& baseUrl)
 {
-   const WebProfile* pProfile = dynamic_cast<const WebProfile*>(profile());
```

Comment:
Since all of our `WebPage` implementations contain our own `WebProfile` implementation this code should be superfluous (let me know if I might be missing something)

Reply:
Yes, I think this should be okay -- when I wrote the `dynamic_cast` I couldn't convince myself 100% that this was the case, but we should know pretty quickly if it isn't!